### ISSUE TITLE
MIP7 - Melonomics

### ISSUE THREAD

**Issue by toms119** [enzymefinance/enzip#7]
**Melon Improvement Proposal**

- MIP: 7
- Title: New Melonomics Proposal
- Authors: [Tom Shaughnessy](https://twitter.com/Shaughnessy119), [Ceteris Paribus](https://twitter.com/ceterispar1bus), [Chris Manessis](https://twitter.com/moonchaincap)
- Status: Draft
- Type: MIP
- Created:25.06.2020
- Reference Implementation:TBD

**Abstract**

The current Melon token economics (Aka Melonomics) are not aligned with the growth of the Melon platform and do not adequately incentivize Melon token holders (MLN). This MIP introduces new Melonomics that enhance the current burn mechanism needed for the MLN token to capture the high growth potential of the Melon platform. Also, the fees to investors and fund managers can be difficult to conceptualize. Our new proposal makes these costs more transparent by using a simpler, easier to understand calculation methodology.

**Background**

Currently, the Melon token (MLN) operates as a mint and burn model. Every year, a set amount of 300,600 tokens are issued and primarily go to compensate maintainers and platform developers working on improving the Melon platform and its user experience. All leftover tokens of the 300,600 that do not get allocated to platform developers are sent to the Melon engine and are subsequently burned. The current issuance schedule is disinflationary as a set number of tokens are issued against an increasing token supply if the token burn does not offset this issuance.

The MLN token is also used as asset management gas, or a fee, that users pay to the Melon platform for the use of its services (on top of ETH smart-contract fees). When setting up a fund, requesting an investment, and executing an investment, users pay amgu (asset management gas unit) in ETH to the Melon network. The number of amgu consumed multiplied by the amgu price equals the amount of ETH users pay for a payable function. For instance, if a fund manager wants to set up a fund, they would currently need to pay ~ 17,500,000 amgu * 0.0000005 MLN (current amgu price) = 8.75 MLN at an exchange of ~0.0570 MLN/ETH =0.498 ETH or $116. All ETH fees are sent to the Melon engine contract which buys MLN/ETH at a premium to the market price and burns all MLN bought every 30 days. The amgu price is set by the Melon Council DAO and can be adjusted based on the network environment. 

Although the amount of MLN tokens burned is directly correlated to the amount of amgu consumed and thus the usage of the Melon platform, the present Melonomics are not directly tied to the growth of AUM in the Melon Protocol and MLN burns are hard to forecast. One of the challenges with the current structure is that it requires the Melon Council to somewhat arbitrarily change the amgu price when they see fit. This makes it difficult for holders of MLN to forecast future burns and for fund managers and investors to forecast costs. Our proposal will simplify this for all those involved, since fees will be set on a % basis and scale directly with AUM. Holders of MLN will be able to forecast potential platform earnings in USD to compute cash flow models, and fund managers will have full transparency over their costs.

**Motivation**

It has become clear to us that there is a genuine demand for a product like Melon. With continuous improvements on the horizon, we expect user adoption to increase exponentially - especially considering the growing attention being paid to the DeFi space. However, without a proper value capture mechanism, the MLN tokenholders, the Melon developer ecosystem and the long-term potential of the platform are all likely jeopardized. Our motivation is to help address these issues before they become a problem by proposing new, more robust Melonomics that are suitably built to capture the value of Melon's growth potential. Our proposed Melonomics model is not only simpler than the existing complex gas model (since most fees are in U.S. Dollar terms), but the proposed AUM based fee to the platform scales MLN burn with the growth of the platform whilst still keeping costs extremely low vs traditional fund cost structures.

**New Tokenomics Overview:**

**Platform Costs**

- 10bps fee on investment (each time an investment is made by a user into a fund, including managers if they invest in their own fund)
- 20bps AUM fee (recurring fee on average daily AUM - computed as 0.05478bps fees per day on daily AUM)
- Fee discount on the 20bps AUM fee for staking/holding MLN. Can be held outside or inside fund at manager discretion, so as not to force an overweighting of MLN within funds:
- Tier 1: 100 MLN, 5% Discount
- Tier 2: 1,000 MLN, 10% Discount
- Tier 3: 5,000 MLN, 15% Discount
- Tier 4: 10,000 MLN, 20% Discount
- Fund managers who hold MLN are exclusively entitled to sell MLN/ETH to Engine at a premium (ie. buy ETH/MLN at a discount)

We have cut the initial fee for a fund manager to set up a fund to zero, since this was a previous user experience hurdle and it does not drive a significant amount of fees to the platform. In our model, a $150 fund set up fee would amount to under 2% of total fees to the platform in year 10 which is not worth the UX hurdle which can impact the growth of funds and users.

All fees will be paid in ETH and sent to the Melon engine which will purchase MLN using ETH through its contract for a premium. All the MLN purchased will be burned.

**Issuance**:

Currently, Melon’s issuance schedule features issuing 300,600 tokens per year, forever. We see value in having issuance as deflationary over time to drive more value to MLN token holders. Our proposed issuance schedule below encapsulates this idea.

- From 2021, token issuance drops 20% per year from 300,600 MLN until it goes ever closer to zero. 
- Any remaining tokens that do not go to developer grants are allowed to be rolled over for one year. For example, if 100,000 MLN is unused for grants after the first year, this can be used for the following year:
- 300,600 MLN issuance for grants in 2020
- 200,600 MLN distributed as grants
- 100,000 MLN left -> gets rolled to 2021
- 2021 issuance = 240,480 MLN
- 2021 MLN to spend = 240,480 + 100,000 = 340,480 MLN
- In 2022, max MLN that can be rolled over is 240,480 (2021 issuance) and any unspent remaining MLN from prior rollovers is burned.

The chart below shows the MLN token schedule over time:

<img width="554" alt="Screen Shot 2020-06-25 at 5 53 11 PM" src="https://user-images.githubusercontent.com/35900106/85799147-c02ebb00-b70c-11ea-9c9d-c0c732b8331a.png">

*Example:**

- Fund manager opens up fund (no charge).
- Investor # 1 invests $50K into fund, pays 10bps ($50) fees.
- Investor # 2 invests $50K into fund, pays 10bps ($50) fees.
- Total fund value is $99,900.
- Manager charges 2% management & 20% performance fee.
- After one year, fund value has increased to $150K.
- AUM fees have accrued on average a daily balance of $125K (assuming linear growth).
- AUM fees = $125K * 20bps = $250
- *If manager held MLN as per tiers above, they would pay a discounted rate.
- Manager earns $125K * 2% = $2,500 management fee.
- Manager earns ($150K-99.9K)*20% = $10,020 performance fee.
- When manager redeems fees, they receive $10,020 + $2,500 - $250 = $12,270.
- Total AUM at end of year = $150,000 - $12,520 = $137,480.
- Investors #1 & 2 positions each worth $68,740.
- Melon engine will have received $50 + $50 + $250 = $350, which will be used to buy & burn MLN.
- Fund manager took advantage of MLN arbitrage:
- At the end of a thawing period (fees accrued from all sources), there are 8 ETH of fees in the Melon Engine and the market price of MLN/ETH is 0.02. The Melon Engine will, therefore, be offering MLN/ETH at a 10% premium enabling fund managers to buy 400 MLN for a total of 8 ETH and immediately sell MLN for a total of 8.8 ETH to the Melon Engine. The user is entitled to 0.8 ETH of risk-free profit.

**Reference Model**

A model of the new Melonomics along with all associated graphs can be found [here](https://docs.google.com/spreadsheets/d/1dMCw4uY8A2LS9OIIDlOxWb3FuoJsuA-mUftWoWf78jA/edit?usp=sharing). 

**Proposed Next Steps**

- MIP7 submitted ✅
- Melon Council discusses and communicates MIP7 proposal on all relevant channels 🟩
- Community feedback period: community can make other submissions/comments over the next weeks 🟩
- Community feedback is taken into account and amendments are made to the proposal  🟩
-Melon Council DAO votes on MIP7 based on community feedback and strength of the proposal 🟩 
-If voted in favour, Avantgarde Finance will then be tasked with implementation in 2020. Any changes that deviate from the core goals of the proposal will be put up for review and voted on again 🟩
-Implementation of MIP7 proposal complete 🟩

**Additional**:

Disclosure: The authors hold positions in MLN and will abide by a no-trade policy for 7 days.

**Comment by FelixOHartmann** [comment]
Great proposal Tom. As both a Melon Fund owner and significant MLN stakeholder, I think this would greatly align interests of all involved, while ensuring the fiscal longevity of the project. 

**My one amendment recommendation is as follows for _Issuance_ :

_- Issuance shall be cut by 50% immediately, reducing it to 150,300 per annum. 
- From 2021, token issuance drops 20% per year from 150,300 MLN until it goes ever closer to zero._**

This will ensure that issuance drops below 10% immediately, allowing burn rate to catch up to it more quickly. At this rate, team will still get rewarded with a bigger stake in the network over the years than nearly any other project out there that is still operational. 

Great work, you have my full support. 

If anyone is interested in contributing to the follow up to this regarding improving governance, contact me on twitter @felixohartmann.

**Comment by ceterispar1bus** [comment]
Some comments were raised around the monthly arbitrage process with the melon engine. Currently, this works as a first come, first served process on a monthly basis where one single manager can take advantage of the arbitrage. To allow a more democratic process, while still keeping it relatively simple, propose the following:

After the 30 day thaw period, there is a 1 day period where fund managers can allocate the amount of MLN they wish to sell to the engine. After the one day period, the engine will sell ETH for MLN on a pro-rata basis to each fund manager that subscribed.

Example:
- 10 ETH in engine at end of month
- Engine is selling ETH for 0.05 MLN (200 MLN)
- Manager 1 wishes to sell 100 MLN
- Manager 2 wishes to sell 100 MLN
- Manager 3 wishes to sell 50 MLN
- Total MLN subscribed to monthly arbitrage = 250 MLN. Arbitrage process is oversubscribed by 25%.

- On day 2, Melon engine sells ETH for MLN as follows:
- Manager 1 sells 80 MLN for 4 ETH
- Manager 2 sells 80 MLN for 4 ETH
- Manager 3 sells 40 MLN for 2 ETH

- If arbitrage process is undersubscribed, the engine opens up for any fund manager **or investor** to empty the engine of the remaining ETH.

**Comment by cojoq** [comment]
Some comments on the proposal:

-  I don’t believe that making fund initiation free will help Aum. It will only lead to more funds that, especially with a limited asset universe, are just more of the same. I’m Against making the fund free therefore, fund managers should have some skin in the game to prevent them from spamming the platform with useless funds. 

- Fees on trades should be limited to gas cost only, in order not to disincentivize active managers over passive managers. Please take into account that gas trading fees are now paid by the fund manager, not the fund, and therefore are costs against management/performance fees received. Ideally, there should be a way to have the gas trading fees paid by the fund. 

- I don’t like the entry fees for investors (10bps). I’d rather see those free, yet the redemption fee higher, say 50bps. Incentive should be to get and keep as much Aum on the platform as possible. 

- long term I would expect more ‘index tracker’ funds to appear on melon. 20bps platform fee (and typical 50bps management fee, no performance) might be a bit too high for tracker funds. So I propose a platform fee that is directly linked to the fund’s management fee. E.g. 10% of the fund’s management fee as a platform fee. 

- it’s important in my opinion that performance fees are NOT shared with melon in any way, other than through increasing Aum and therefore the base of management fee share. Not that this is now proposed, but if it would I’d argue against it. 

- to support Aum growth it would be helpful to have some sort of referral system for open funds. If you bring investors to the fund, you receive compensation, e.g a revenue share of the management fee for a limited period of time. This could also be a source of additional revenue for melon and third parties.

**Comment by fubhy** [comment]
> I don’t believe that making fund initiation free will help Aum. It will only lead to more funds that, especially with a limited asset universe, are just more of the same. I’m Against making the fund free therefore, fund managers should have some skin in the game to prevent them from spamming the platform with useless funds.

That is something that could be solved on the frontend instead of the protocol by adding clever filtering and sorting strategies that make these "spam" funds technically "invisible" by ranking them extremely low. To add "skin in the game", the fund manager would then have to invest into their own fund initially to cause the fund to be ranked and visible. Through this approach, we still remove the barrier for fund creation and allow newcomers to experiment with the network without a "large" (I know that this is relative) upfront cost. And without the frustration of accidentally breaking your fund through too restrictive policies and then having to set up a new fund allover again with the same, relatively high cost.

> Fees on trades should be limited to gas cost only, in order not to disincentivize active managers over passive managers. Please take into account that gas trading fees are now paid by the fund manager, not the fund, and therefore are costs against management/performance fees received. Ideally, there should be a way to have the gas trading fees paid by the fund.

I am also not a fan of additional per-trade network fees. Both because this would, as you said, penalize active fund managers, but also because it would make it less appealing to develop e.g. trading bots on top of your fund. One approach that we discussed internally was to collect fees by piggy backing on top of the configured fund manager's fees (your argument against taking a cut of the fund managers performance fee could be solved by instead collecting it based on the AUM exclusively).

> I don’t like the entry fees for investors (10bps). I’d rather see those free, yet the redemption fee higher, say 50bps. Incentive should be to get and keep as much Aum on the platform as possible.
That is a very interesting idea and very much inline with the idea of removing the barrier to fund creation by eliminating upfront costs.

**Comment by ceterispar1bus** [comment]
> * Fees on trades should be limited to gas cost only, in order not to disincentivize active managers over passive managers. Please take into account that gas trading fees are now paid by the fund manager, not the fund, and therefore are costs against management/performance fees received. Ideally, there should be a way to have the gas trading fees paid by the fund.

I think there is a bit of a misunderstanding. We're not proposing any fees on trading. The 10bps is on the funding transaction only, not "investing" in assets after the fact. Anything within the fund just incurs standard Ethereum gas.

> * I don’t like the entry fees for investors (10bps). I’d rather see those free, yet the redemption fee higher, say 50bps. Incentive should be to get and keep as much Aum on the platform as possible.

We discussed this a bit when modelling out but kept fee same. Would be open to no investing fee but higher on exit. Would cause assets to stick more and increase earnings over time.

> * long term I would expect more ‘index tracker’ funds to appear on melon. 20bps platform fee (and typical 50bps management fee, no performance) might be a bit too high for tracker funds. So I propose a platform fee that is directly linked to the fund’s management fee. E.g. 10% of the fund’s management fee as a platform fee.

Hard to do this because managers can set whatever management fee they want, even 0%. This would cause $0 fees being paid to protocol. Managers would just set 0% management fees and a higher performance fee.

**Comment by Deepcryptodive** [comment]
Great initiative!

I agree with most of these changes, except that the proposed **new AUM fee** is currently to be payed by the Fund Manager. This is unfair to Fund Managers that now charge little to no management fees.  

I propose to either: 
1. Make the Fund itself pay the fee (e.g. from the assets contained within)
or
2. Allow a one-time change in management fee

This will avoid some managers to be treated unfairly, and prevent the incurrance of net losses.

-----------------------------------------------------------------------------------------------------------------------------

One may also reconsider whether **trading fees** should be best payed by the manager, or by the fund itself. With the current gas prices this is a heavy cost for active fund managers.  As cojoq pointed out: "Ideally, there should be a way to have the gas trading fees paid by the fund". Although I am not sure how to best implement it. Preferably, it should avoid that malicious (or incompetent) fund managers are able to drain a fund via a bunch of useless trades. Maybe only allow trading fees to be claimed by a manager, if they result in a net profit? E.g. add the net on-chain trading costs (which were payed in ETH by the manager) to the performance fee payout, and allow claiming of them similar to the performance fee. Only allow claiming if there was a net profit in that performance fee period.
This would incentivize good trading behavior, as well as avoid managers having to guess what their optimal performance fee should be (which could otherwise result in managers settin very high performance fees, to account for an environment with very high gas prices). 

-----------------------------------------------------------------------------------------------------------------------------

Finally, I support ceterispar1bus's proposal concerning subscription for the monthly arbitrage process. Currently it is very difficuly to swap MLN to ETH via the melon engine.

**Comment by toms119** [comment]
Thanks for all the comments everyone we are reviewing them all

I'd tend to agree that the fund paying the protocol based AUM fees would make more sense than pulling it out of man fees + performance. Will think this over.

**Comment by cmanessis** [comment]
@Deepcryptodive, thanks for your comments! 

Regarding: " I agree with most of these changes, except that the proposed new AUM fee is currently to be payed by the Fund Manager. This is unfair to Fund Managers that now charge little to no management fees".

So, fund managers will not pay for this fee, but instead, it will be paid collectively by all funds in the following manner:

20bps AUM fee (recurring fee on average daily AUM - computed as 0.05478bps fees per day on daily AUM)

I hope this makes sense.

**Comment by FelixOHartmann** [comment]
> Some comments on the proposal:
> 
> * I don’t believe that making fund initiation free will help Aum. It will only lead to more funds that, especially with a limited asset universe, are just more of the same. I’m Against making the fund free therefore, fund managers should have some skin in the game to prevent them from spamming the platform with useless funds.

Pay-walls are known to throttle on-boarding. It's free to create an e-trade or binance account, so it should be free to set up Melon too especially since a first set up may result in mistakes and need a restart. Someone else mentioned skin in the game. Instead of charging $100 in fees, we can make it a minimum deposit of $100 to launch. No unfunded funds. 

> * Fees on trades should be limited to gas cost only, in order not to disincentivize active managers over passive managers. Please take into account that gas trading fees are now paid by the fund manager, not the fund, and therefore are costs against management/performance fees received. Ideally, there should be a way to have the gas trading fees paid by the fund.

Neither currently nor in the proposal are amgu charged for trading. Could be interesting charging trading fees to fund though especially should someone employ a more high frequency approach down the line. 

> * I don’t like the entry fees for investors (10bps). I’d rather see those free, yet the redemption fee higher, say 50bps. Incentive should be to get and keep as much Aum on the platform as possible.

10bps on a $1000 investment is $1. That is less than you pay on coinbase, kraken, or anywhere else to buy a coin. 

> * long term I would expect more ‘index tracker’ funds to appear on melon. 20bps platform fee (and typical 50bps management fee, no performance) might be a bit too high for tracker funds. So I propose a platform fee that is directly linked to the fund’s management fee. E.g. 10% of the fund’s management fee as a platform fee.

Both use the same technology. If someone charges higher management, they may also have higher expenses, so it should be flat independent if someone charges 0 or 5% management. Not to mention some will just opt for high performance and no management and pay 0 with such a model. 

> * to support Aum growth it would be helpful to have some sort of referral system for open funds. If you bring investors to the fund, you receive compensation, e.g a revenue share of the management fee for a limited period of time. This could also be a source of additional revenue for melon and third parties.

Alternatively allocate an incentive pool from mint and you receive kickback for any funds that stay on Melon for 6 months. Let the yield farmers come. 


> I agree with most of these changes, except that the proposed **new AUM fee** is currently to be payed by the Fund Manager. This is unfair to Fund Managers that now charge little to no management fees.

Do keep in mind that in the traditional world you pay 50-100k for the most bare bones fund set up. If 20 basis points on AUM break a manager they should probably not be running a fund. That said, perhaps the setting can be changed that there is no 0% management but that minimum is set to .2% (20bps) and if not raised, all of it will go to Melon Engine.

**Comment by ceterispar1bus** [comment]
> Alternatively allocate an incentive pool from mint and you receive kickback for any funds that stay on Melon for 6 months. Let the yield farmers come.
> 

Can you expand a bit on this? We've thought a bit about yield for MLN holders but couldn't think of anything thoughtful. I don't think we should add dilutive inflationary rewards right now just because they're in vogue, but open to other opinions.

**Other thoughts:**
1. Re-thinking the AUM fee - I do believe it makes more sense to have the fund pay this in-kind than the manager from their fee income. At the end of the day, whether fund managers pay the full fee or not, investors will still end up paying the fee directly or indirectly. If we did it this way, I think we should scrap the discount tiers for a manager holding MLN. First, why would a manager subsidize the lower fee rate for everyone? The lower fee would most likely be passed on anyways in the form of higher mgmt/perf fee. Second, the discount fee savings would probably not be worth the opportunity cost of locking up that capital.

2. For the arb process, I think we institute that the MLN used for the arb must be held at least 15-30 days. This would prevent managers from just buying MLN on day 30 to sell 1 day later (high token velocity). Also, when lending protocols implement MLN, managers would be able to borrow -> arb -> repay in a one day cycle. Implementing a minimum MLN hold period to take advantage of the arb would prevent these activities, while potentially resulting in a higher yield for lending MLN as well.

**Comment by cojoq** [comment]
> Hard to do this because managers can set whatever management fee they want, even 0%. This would cause $0 fees being paid to protocol. Managers would just set 0% management fees and a higher performance fee.

One way to handle this:

IF performance fee > 0 and management fee <= 200bps, THEN platform fee = 20bps;
ELSE performance fee = 10% x management fee.

This way you'll be able to accommodate tracker funds long term imo.

**Comment by cojoq** [comment]
> 10bps on a $1000 investment is $1. That is less than you pay on coinbase, kraken, or anywhere else to buy a coin.
> 
> >

Technically, investors have already bought coins in order to participate in the fund. Therefore, they would incur trading fees twice (apart from today's additional silly gas fees), which is suboptimal imo if the goal is to attract new AuM. I agree that 10bps is not an issue for small investments, but it will become an issue for bigger ones, as they typically seek lower (less risky) returns. Also, as fund strategies expand from the typical long crypto volatility to e.g. dedicated stable coin yield / liquidity providing, then these fees do start making a substantial difference on investor returns.

> Through this approach, we still remove the barrier for fund creation and allow newcomers to experiment with the network without a "large" (I know that this is relative) upfront cost.

> Pay-walls are known to throttle on-boarding. It's free to create an e-trade or binance account, so it should be free to set up Melon too especially since a first set up may result in mistakes and need a restart.

Fundamentally I believe that Melon's USP is that it allows you manage other people's crypto for a fee. If you just want to manage your own crypto, there are numerous alternatives available. As said by others, the cost for setting up a Melon fund compared to a CeFi fund are next to nothing, even at $100. I would seriously consider _increasing_ that onboarding fee for production funds in due term once functionality of the platform is on par. 

Besides that, I think it is false to assume that you will attract more AuM if you make it less costly to create a fund. The focus should be on making it easier to onboard _investors_ as they are the long-term revenue generators. MLN holders should theoretically be indifferent which funds hold the AuM, as long as the crypto remains on platform. 

For testers I'd consider a testnet such as Ropsten, all free of charge. Currently, test funds (like the one I created before going into production) are just sitting there unused, clouding the production network statistics.

**Comment by toms119** [comment]
Felix,

Appreciate your comments on the proposal!

> Pay-walls are known to throttle on-boarding. It's free to create an e-trade or binance account, so it should be free to set up Melon too especially since a first set up may result in mistakes and need a restart. Someone else mentioned skin in the game. Instead of charging $100 in fees, we can make it a minimum deposit of $100 to launch. No unfunded funds.
> 

I do like this idea. Especially since the manager becomes an LP at that point and barriers to launch a fund are lower

**Comment by toms119** [comment]
> 
> 1. Re-thinking the AUM fee - I do believe it makes more sense to have the fund pay this in-kind than the manager from their fee income. At the end of the day, whether fund managers pay the full fee or not, investors will still end up paying the fee directly or indirectly. If we did it this way, I think we should scrap the discount tiers for a manager holding MLN. First, why would a manager subsidize the lower fee rate for everyone? The lower fee would most likely be passed on anyways in the form of higher mgmt/perf fee. Second, the discount fee savings would probably not be worth the opportunity cost of locking up that capital.

Agreed the discount wont equal the opp cost of locking up the cap, and its annoying to mentally think about for a manager in their risk profiles

> 2. For the arb process, I think we institute that the MLN used for the arb must be held at least 15-30 days. This would prevent managers from just buying MLN on day 30 to sell 1 day later (high token velocity). Also, when lending protocols implement MLN, managers would be able to borrow -> arb -> repay in a one day cycle. Implementing a minimum MLN hold period to take advantage of the arb would prevent these activities, while potentially resulting in a higher yield for lending MLN as well.

Agreed on the holding period. Then again doubt fund managers will want to unload a day later if MLN is underwater since then they lose the arb gains so they may hold. I like the hold to prevent the flash loan idea. Agreed there.

**Comment by MouyouMoos** [comment]
I would link the token to the performance of the funds. Thats what melon stands for. make more profit with melon! when people earn more they are willing give more to the infrastructure which made it possible. don't charge the investors, charge the portfolio managers on their profits they made because of melon. that makes much more sense.

**Comment by MouyouMoos** [comment]
Also I would like to see a pre vote of melon token holders on proposals, fund distributions and other votes. this vote doesn't count it just the council can decide but it shows what the token holders want and what the council decides. it makes the process more transparent and shows the alignment / misalignment between council and token holders.

**Comment by cojoq** [comment]
Hi all - some feedback after the community call:

In my view it's good _in general_ to have a clear business model for the protocol which should make it easier to fundamentally value the token, and with that attract liquidity to it*. 

My question is though, assuming the shared goal is to enable the long tail in fund management, if a basispoint AUM fee should be preferred over a fixed fee per fund. If we take today's numbers ($1.5m AUM and 360 funds), the protocol could expect $3000 in yearly revenues at a 20bp protocol fee, of which 80% would be paid by the top 10 funds. It's difficult for me to see why the (investors in the) top AUM funds should be subsidising the (feature enhancements for the) micro funds. 

Alternatively, if every fund was charged a yearly $10 fixed 'maintenance' fee, the protocol would earn more. So the key question I would pose is: **are we focussing on growth in funds or growth in AUM**? Long tail/friction comments suggests growth in funds over growth in AUM, in which case a fixed yearly fee per fund should make more financial sense. In my view a yearly maintenance fee of $100 per fund would still be peanuts btw, less than a Netflix subscription ;) At the same time such a fixed fee could entice managers to come up with more AUM to give it a serious go, instead of the plethora of micro funds currently on the platform with less than $1000 AUM. In order to avoid being called an asocial capitalist pig: the council could always wave the fee for some micro funds with a heartbreaking story. 

Should the community decide that AUM growth is to be preferred over fund growth, i.e. capture the 'mid tail' of funds that currently can't make it in CeFi due to the $50K+ yearly cost, but do want to launch a somewhat sizeable melon fund, then a basispoint fee would make more sense imo long-term, provided that the fee structure can cater to index funds (or yield funds for that matter), which typically operate between 30bp-75bp management fee and no performance fee, and contain most of the fund market in CeFi, so why not in DeFi.  Also, the larger AUM funds should then be getting some privileges (which ones?) for carrying most of the protocol fees imo. 

Of course, there's also the possibility to combine the two models above: an Aum fee, with a minimum of $x per year. 

Looking at the proven traction thus far in melon, I would seriously consider opting for the fixed fee model, be it on a temporary (introductory) basis while leaving open the option that an Aum based fee (with higher minimum) could be applied in future. That leaves more time to figure out where the (new) sweet spot for melon funds is in the market and the corresponding optimal business model, while putting the focus on growth in funds. 

Finally a practical point on how a new fee model would be handled. I assume the protocol fees would be reserved, then regularly claimed in shares of the fund, redeemed in kind, and finally traded to MLN and burned? If that's the case then I hope that process can be automated (including the required whitelisting of the melon address in closed funds).

*On a side note: rule of thumb CeFi asset management valuation (https://mercercapital.com/riavaluationinsights/valuation-of-asset-management-firms/) at 50% margin for token holders (and 50% for development cost) would value the protocol at approx 100bp of Aum. At $22m market cap, the token is  currently pricing in future average AUM of $2.2b. Expect some analysts to scratch their heads with less than 0.1% of that realised today. So I'm not really convinced that such 'easy to understand tokenomics' -if and when approved- will be beneficial for those holding MLN bags right now.

**Comment by levin-ilya** [comment]
I like the proposal and hope some version of it passes

**Comment by dennygalindo** [comment]
One more quick comment I thought of while debating this on telegram. 
The idea to charge a fee on the way in and way out is very similiar to maker. You accrue interest continuously but only pay (and burn) when you close out. Makers policy has been the worst among defi at linking token value to adoption. This proposal goes down the same path. It’s much better to create continuous demand for melon. This is best done by requiring fund managers to hold an amount of melon linked to their aum. This continuously links aum and melon market cap. The requirement could be implemented  by forcing melon into all funds (say 1 pct) or by forcing melon to be held on a second account by the fund manager (more like uma). Requiring fund managers to hold melon to run their funds would turbocharge melon like the Snx move turbocharged Snx. We could pay fund managers who grew aum with freshly minted melon if they increased aum fast enough (say 10 pct a week) . It’s so easy to bootstrap this project with this easy incentive change.

**Comment by FelixOHartmann** [comment]
Some interesting comments since we last discussed in June.

Here's a revamp I think could make sense:

Allow tail-end, but charge a small 'listing fee (e.g. $10 - $100/year) to get listed on terminal. 
On one hides the small/dead funds. On the other it creates further earnings. 

Inspired by what @dennygalindo wrote, we could give fund managers two options: 
Pay a fixed BP AUM fee (which then buys and burns MLN), just like traditional fund administrators do, or have them hold 2% of their fund in MLN to wave fees. Balance has to be average daily balance so funds don't just bulk buy and sell at a snapshot date. 

Hence either supply burn or supply sink.

**Comment by dennygalindo** [comment]
I like the option idea. (Though it’s more complicated). The snapshot issue is not that important in my view. If you must be in compliance to make a trade or other action. then fund managers will fill up with melon when they want to do something.

**Comment by perhamgirl** [comment]
[Melon Council comments on MIP7 
](https://medium.com/melonprotocol/melon-council-comments-on-mip-7-daa58b8a4554)

**Intro**
The Melon Council has been working on assessing the viability of Melon Improvement Proposal #7 (MIP#7) and would like to share some initial feedback on the proposal submitted on June 25th, 2020 by Tom Shaughnessy, Ceteris Paribus and Chris Manessis (Moonchain Capital).

**What is MIP7?**
As a reminder, the proposal takes a closer look at the Melon token economics (Melonomics) and lays out arguments to reconsider the value accrual mechanism. In particular, it argues for a value accrual mechanism that is directly linked to growth in the Assets Under Management in Melon Protocol funds vs. the number of individual Melon Protocol funds. Crucially, it also argues for removing any fee at fund “set up” to entirely remove any friction for new users entering the platform and deploying a Melon fund on the blockchain. It also proposes to give holders of MLN discount to fees on the protocol and the argumentation to commit towards reducing inflation in later years.

**Removal of the set-up fee**
Our internal analysis has shown that for the set-up fee to accrue any real value to the network over time - fund set-up fees would have to go up. This would only serve to increase the barrier to entry which would be out of line with our fundamental goal to empower and enable the long tail of managers and their investors.
Therefore, it makes sense to remove the set-up fee if there is a viable alternative that can better align stakeholder interest.


**Moving towards an AUM based fee**
MIP#7 also puts forward a view for a 20bps recurring fee on AUM.  In the last year or so, protocol development has come dangerously close to halting because of a token price that was too low to sustain the maintenance and development costs. It is only because of the personal capital (either in the form of investment or salary reductions) risked by contributors to the ecosystem that Melon is still around and thriving as a project today.

We acknowledge that Melonomics is probably at least partially to blame for some of the challenges the protocol faced in the last year. In order to accrue value on a forward-looking basis with the current model we would need to increase the AMGU price substantially from where it is now. Given that Melon’s core proposition has always been about empowering the long-tail, we feel that it would be wrong to stunt up-and-coming managers by providing such a barrier to entry. Our preference is to grow with our users. If they are successful, Melon will be successful.

At the same time, we want to emphasize that the single most exciting thing about Melon is that it empowers the long-tail of fund managers and investors in asset management - a market that has never been served before.

With that in mind, if the choice between accruing relevant value to the network is between increasing the set-up fees in the order of 10-15x or imposing a minimal (20bps) AUM-based fee would be in favour of the latter, which allows the network success to grow alongside user success. 

Having said that, the Council is in favor of an AUM-based fee but will not exclude the option of giving a choice to the user of either an AUM based fee or a much larger “set up” fee, which can each help achieve the same value. 

**Caveat**: There has been some discussion over whether an AUM-based fee should be charged to the fund or to the manager. We believe this topic should be explored further, and it is not something we reached consensus on in our initial meetings on the topic. 


**Fees on investment**
A fee on investments seemed to be taken negatively by the user community. There have been some suggestions to remove this altogether and replace it with a higher fee on redemption and the Council thinks this could be a better direction. 


**Holding MLN as entitlement to a discount on the network**
In principle, we like this idea very much, but before committing to something like this we want to explore other routes, such as for example; referral programs. We are currently researching these alternatives and others that have been suggested by the community and internally in our discussions.


**Reducing Maximum Inflation**
It is hard to make predictions about when future spending might peak. The priority is to get the protocol in a mature and stable state before formally committing to reductions in inflation. Some signs of that would be:
Less frequent protocol upgrades
Reaching 10,000 users
Reaching $1bn in AUM

With that in mind, we want to confirm that the Melon Council is committed to reducing the available inflation pool when such signs are visible. We can commit to re-accessing the above metrics and communicating regularly on our  thinking if it changes. The Council does not carry tokens over from year to year. If there are unspent tokens left in any given year, they are typically burnt.

**Concluding Comments**
In summary, the Melon Council supports all the general ideas in MIP#7. The exact details on how these get applied will depend on further analysis, feedback and input from user representatives on the Council and a technical assessment that will take place before implementing these details.

**Comment by amadeobrands** [comment]
I have created the first draft of an exploratory phase document that explores Token Economic Design Directions (TED) for the Melon Protocol and explores ways to improve Melonomics. After this phase I would suggest the following steps towards creating new Molonomics:
- Completing the Exploratory phase
- Setup new Melonomics
- Feasibility Study/Market Testing
- Technical implementation plan

[Melon Port _ Exploratory phase - Token Economic Design.pdf](https://github.com/melonproject/MIP/files/5229564/Melon.Port._.Exploratory.phase.-.Token.Economic.Design.pdf)

**Comment by dennygalindo** [comment]
I am worried about the idea that fund managers could pay the cheaper of a fixed fee and aum fee. This will result in less demand for the token/ fees burnt because funds will choose the cheapest option. 

I think you could have a small fixed fee AND an aum fee to prevent funds that are super small from clogging up the system. 

There were some comments above about an aum based fee of 20 bps. While this is reasonable you actually create 5 times more demand for melon with a requirement to stake 100 bps of aum. This is actually less like a fee and more like a combination of a deposit and equity stock option reward. A fund that closed down would get their melon back. They might lose or might gain depending on if the platform does well. This actually incentivized them to evangelize to other pms! It’s not zero sum. All fund managers want as many funds on the platform so their deposit (stake) will appreciate. They are incentivized to bring in new money to the platform even if they don’t win share. This is much better performance. 

Crypto works best when it’s thought of as a protocol not a company. A company would charge fees but a protocol rewards employees with options of the protocol is adopted. A traditional  fund platform might pay people based on their performance but they also get compensated on the fund family’s  (trow, alliance Bernstein) performance through stock options. A staking mechanism breaks out of the company mindset and designs a protocol token to coordinate somewhat adversarial participants towards working together much like miners work together to get adoption even while they fight for market share. 

One other benefit is that the token price actually is a marketing mechanism. How many people try something out after they see the token shoot ico 50 pct. a staking mechanism gets 5x the buying that a 20 bps aum fee would get and would lead many to explore the platform. It’s also front loaded so you don’t have to wait a year to collect. 

Economically speaking if crypto takes off over next 15 months a  1 pct stake requirement is actually no fee at all while a 20 bps fee in ethereum could turn out to be a 2-4 pct
If we look back from dec 2021. Our fund managers who are optimistic about crypto would surely assume that parting with ethereum now if a difficult trade off. They would much rather hold something they can get a refund on if they dislike the system or find it too hard to be a successful fund manager.

I hope a staking requirement based on aum is implemented 

1) best alignment of incentives to grow platform aum
2) biggest impact on price of token which might cause a feedback loop to more aum
3) lowest friction (maybe you make money on being an early adopter even if you quit) 
4) simplest to implement. I imagine you can copy code from Snx and dev can still be easily funded through inflation. 

Thanks for listening.

**Comment by amadeobrands** [comment]
If funds are <= 30K USD pay yearly listing fee else pay a fixed BP AUM fee (which then buys
and burns MLN), just like traditional fund administrators do, or have them hold X% (Depending
on AUM) of their fund in MLN LP tokens to wave fees. Balance has to be average daily balance
so funds don't just bulk buy and sell at a snapshot date.

**Comment by rusnewton** [comment]
MLN – MIP7 Comments

By way of background, I have been in the trading business since 1987 and co-founded a commodity hedge fund in 1999. CoinShares (which I also co-founded) has been active in the crypto space since 2013.

When considering whether to amend the MLN tokenomic model, I believe it is worthwhile looking at the traditional asset management space and trying to understand what part Melon might play in that in future. Whilst there may be $4T invested in various hedge fund strategies typically paying up to 2% management fees & 20% performance fees, some surveys suggest there is more than $20T in actively-managed "traditional" strategies plus another $10T+ in passive strategies. Both these latter categories will attract much lower fees (and normally no performance fee at all) but collectively represent a far larger TAM. I believe that much of the thinking behind the proposed revisions to the MLN tokenomic model is predicated on the idea that most if not all of the assets will be actively managed; but actually, it's easy to see why this might not be the case: simple baskets (e.g. of top 10 governance tokens, rebalanced monthly) are easy to construct and represent an attractive way for investors to get broader exposure to crypto and may turn out to be a "gateway drug" for institutions to move towards actively-managed strategies.

I am not entirely convinced that removing the setup fees entirely makes sense. In many instances we have seen situations where the lack of a small up-front charge has led to the creation of frivolous accounts being created. If Melon removes the setup fees, is the Melon Council willing to have thousands of funds showing that hold asymptotically small AuM but cause computational drag and behave like litter in reporting? One way to address this might be through, say, staking MLN into a fund so that the stake is returned when the fund is wound up - thus reducing the clutter of old funds that have been left to die on the platform which might otherwise result from removing the barrier to entry entirely.

It might be complicated to implement something like this, but I wonder if perhaps a distinction could be made based on the fee model that the manager chooses when they first set up the fund:
•	If they charge 2% management fee + 20% performance fee, then fund admin fees will have little impact
•	If the fees are much lower and there's no performance fee, then fund admin fees will have more impact

So maybe you have two pricing models based on whether or not a performance fee is being charged?

Even with passive funds you can still charge an AuM-based fee but it might need to be lower.

It is worth exploring what other potential uses Melon Terminal might have beyond simply replacing the existing hedge fund administration layer in traditional finance. For example:
1.	Many aspiring investment managers need to present an audited track record to investors before they can attract capital; Melon allows them to set up and manage very small amounts of money with very low slippage and ought therefore to attract some of the same sorts of managers that present their ideas on copy trading platforms.
2.	Individuals managing their own money may find Melon Terminal a useful way to manage their crypto assets in the same way that many private investors make use of "wrap" platforms - especially if there were some sort of integration with tax/accounting/reporting apps. 

With regards the 10bp fee proposed on new investments, it is worth noting that if Melon seeks to attract institutional investors to the platform as crypto goes mainstream, then transaction-related fees are likely to be heavily resisted by institutional investors. Again perhaps if it’s possible to engineer a two-speed model (call it “active” vs “passive”) then transaction fees can be higher in the active model as they will be less frequent, but should be very low in the passive model to increase competitiveness.
Adding an AuM-related fee would be a very attractive approach for Melon to take from a tokenomic/valuation perspective; however it is worth bearing in mind that generally speaking, technology platforms like Melon should be trying to undercut the existing offerings. Most hedge funds are paying up to around 12bps in fund administration fees; the main problem for the "long tail" of small funds is the very high minimum fees that are charged (typically $2,000 - $10,000 per month) which "price out" small managers or experimental funds. If Melon introduces an AuM-based fee then it really needs to be lower than what's available for medium-to-large funds on a basis points scale, as well as having no minimum so that small funds have access. I would therefore recommend a fee no higher than 5bp-10bp ($50,000-$100,000 per year for a $100M fund).

I think it would also be wise to consider tiered AuM-based fees so that they become very low as the fund becomes very big - that's typically what a traditional fund administrator would do. Something like the first $125M at 10bp, the next $125M at 7.5bp, the next $250M at 5bp, everything above that at 2.5bp?

**Comment by perhamgirl** [comment]
> If Melon protocol grew to a certain size, where the engine pool was >$1MM monthly, could see MLN token demand an attractive yield to lend. This is one way yield would be incorporated into the MLN token without it being explicit in the design.
> 
> *EDIT - expanding on this last point. Let's say the engine is selling ETH for MLN at a 10% discount every month. Over a year, that's a 120% yield - quite lucrative. At the same time, the monthly arb would probably be oversubscribed every month if this were taking place. A manager may not want to buy and hold MLN because it locks up capital. Instead, they could borrow it purely for the arb play. This would in effect pass some of the yield from managers to lenders. If there was no required holding (ie escrow) period to use MLN in arb, then managers could just complete this in one day, and the lending yield would be low. Requiring a fixed time period to use MLN in arb would stop people from buying (or borrowing) and dumping in one day, reducing token velocity, and shifting yield to lenders, if this so happened to become an active market/strategy.
> 
> *EDIT 2(July 6) - With the announcement that Aave is implementing credit based loans, the borrow/arb strategy will become much more viable for fund managers in the future, reaffirming my thoughts around hold period to take advantage of arb.

**Comment by perhamgirl** [comment]
Hi All, now that we've delivered Enzyme v2 to the auditors we've had some time to reflect on the implementation of MIP7 within the almost entirely new Enzyme framework/architecture. Here are a few short thoughts and considerations to bare in mind based on some of the feedback we've read above and discussed at length with the Melon Council.

**Possible challenges with implementation:**

- Difficult for the protocol to make fee-based calculations based on the management and performance (this idea was previously proposed in this thread)
- It's much more complicated to rely on a fee which depends on once per year for example because there are ways to game it (eg. invest and redeem at opportunistic times just before fee is charged, etc)
- Collecting any protocol fee based on a management fee or performance fee would make things extra complicated since Manager fees are paid in shares of their own fund (much higher gas, complexity to liquidate into MLN etc).

**Proposed more elegant implementation**

- Easier to impose an upfront fee which gives you certain access rights. Eg. If you pay MLN (or the currency which the fund is denominated in) you get X month of usage. This fee can still be calculated based on AUM.
- Think of it as a subscription period or fee where manager loses access to some of the key core functionality (eg. trading) if the fund manager has not paid the fee but other functionality (eg. redemptions) can stay in tact.
- Fee could be implemented in several ways: Eg. Fund manager could pay fees from vault or XYZ could get a free period of subscription if they win a competition, etc.
- Fee discounts could be accessible to funds above a certain size

**Comment by nottrunner** [comment]
I like the idea of a tiered subscription model tied to AUM ala @rusnewton 's suggestion.  

I also think there should be a minimum investment amount, probably as high as $1000 to deter spam funds.  I think its hard to be all things to all people, and the minimum fee should also be used as a baseline filtering mechanism to ensure the platform attracts a high percentage of engaged users.

Enzyme seem to and should skew towards attracting professional and aspiring professional managers. There are other platforms out there that target and would always be more appealing to the retail/experimental user.

**Comment by perhamgirl** [comment]
> I like the idea of a tiered subscription model tied to AUM ala @rusnewton 's suggestion.
agreed, we'll look into what can be done on this front

> I also think there should be a minimum investment amount, probably as high as $1000 to deter spam funds. I think its hard to be all things to all people, and the minimum fee should also be used as a baseline filtering mechanism to ensure the platform attracts a high percentage of engaged users.

I am not sure about this
 
> Enzyme seem to and should skew towards attracting professional and aspiring professional managers. There are other platforms out there that target and would always be more appealing to the retail/experimental user.

enzyme is an infrastructure which people can build retail or professional services on top of. it is definitely skewed to a more professional audience but ultimately the use-case can be determined at a front end level and our hope is that people will white label it and build different use-cases or interfaces on top of it over time (eg.rari)

**Comment by Deepcryptodive** [comment]
Small update for those who missed the latest [community call](https://www.youtube.com/watch?v=VpXkH2pEebU) or the 'Sulu' [blog post](https://medium.com/enzymefinance/announcing-sulu-enzymes-next-release-roadmap-5c754b8e9f42).


> Work on MIP7 has begun. The first phase is to implement a simple based AUM fee as approved by the Enzyme Council back in September and later endorsed by the user-representatives towards the end of last year. This implementation will continue during the next few weeks whilst in parallel exploring other suggestions made for a possible phase II for the tokenomics including staking, referral fees and more.

Slide from community call:
![image](https://user-images.githubusercontent.com/61325205/110940786-bbac4000-8337-11eb-9ae7-5cd89e3e9d4a.png)


Also:
_In Phase I_
- The AUM fee will be payable in the denomination currency (and potentially other liquid assets) to a new type of Melon Engine
- The Melon Engine will convert the collected fees to MLN and burn
- Enzyme Vault managers who hold MLN are entitled to trade to the Engine at a discount (this discount will be determined by the council - but is adjustable over time)
- A minimum fee might be introduced for low AUM funds (<$150k), to avoid larger funds are over-subsidizing smaller funds
- Fee on investments (i.e. when depositing into a Vault) won't be re-introduced 



_In Phase II_

- We'll also revisit reducing MLN inflation in this second phase. Personally, I am in favor of a yearly >20% cut in inflation, although there was no consensus on this (yet) in the Council. Do note that for now, all leftover MLN that is not used for developer grants, will keep on getting burned (and not rolled over to the next year). 


All still very much in agreement with https://github.com/enzymefinance/ENZIP/issues/7#issuecomment-751828454

**Comment by GMT-fintechv1** [comment]
Greetings! I recently learned of Enzyme from a group of crypto hedge fund managers and proprietary traders I know. I have since read up on the protocol and even purchased some MLN.

I'll start with a quick introduction. In 1998, I co-founded [BUYandHOLD.com](https://www.barrons.com/articles/SB95081209910354073), which grew into the 10th largest online brokerage in the country and held the patent for dollar-based investing into equity securities. After riding the dot com bubble up and down, we were bought by Oppenheimer & Co. In 2002, I assumed the role of head of marketing at Oppenheimer where I oversaw the corporate rebranding effort, worked with our asset management team to promote their investment products through our financial advisor network and crafted a marketing campaign to help our financial advisors attract new clients. Having gotten more familiar with the world of hedge funds, I left Oppenheimer in late 2005 to co-found [VanthedgePoint](https://www.finextra.com/pressarticle/7800/vanthedgepoint-middleware-targets-small-sized-hedge-funds), an integrated hedge fund services firm that catered to small and start-up hedge funds.

The industry deemed us a “mini prime broker,” and we embarked on a journey to create not only a new prime broker from scratch, but also a firm that would provide the institutional-grade support necessary for small and start-up hedge fund managers to attract investors and grow their business. We learned a lot before the 2008 financial crisis shuttered the firm, and now that I’ve read through all the comments on MIP7 I thought I’d share my thoughts. Better late than never.

Based on the responses above, I identified three broad goals for the protocol: 

**1. Grow AUM/Grow Investor Base**
**2. Grow Number of Funds/Customer Base**
**3. Grow protocol revenues**

I selected several salient quotes from the responses to MIP7 related to each, and provide my suggestions below the grouping rather than reply to each specific quote.

**1. Grow AUM/investors**

> The focus should be on making it easier to onboard investors as they are the long-term revenue generators. MLN holders should theoretically be indifferent which funds hold the AuM, as long as the investors remain on platform. (@cojoq)

> So to tie melon tokenomics with adoption. All managers must hold -1 percent (or some other amount) of aum in melon. That’s all you need. When they attract more assets they buy more melon if aum goes up faster than inflation (aka development costs and liquidity rewards) then melons value goes up.(@dennygalindo)

> The inflation can be used for 1) encouraging people to put money in aka yield farming 2) development 3) encouraging managers to start funds. Voila viral.(@dennygalindo)

> Crypto works best when it’s thought of as a protocol not a company. A company would charge fees but a protocol rewards employees with options of the protocol is adopted. A traditional fund platform might pay people based on their performance but they also get compensated on the fund family’s (trow, alliance Bernstein) performance through stock options. A staking mechanism breaks out of the company mindset and designs a protocol token to coordinate somewhat adversarial participants towards working together much like miners work together to get adoption even while they fight for market share (@dennygalindo)

Both cojoq and dennygalindo make excellent points. 

Enzyme provides a robust solution for aspiring fund managers from a fund management perspective, just like we did at VanthedgePoint. However, we quickly realized that to succeed as a business, the total AUM on our platform had to grow, which meant that we needed to help our clients grow the AUM for their funds. We decided to create a Capital Introduction platform to present clients to prospective investors such as Funds of Hedge Funds, Pensions, Endowments, Family Office and High Net Worth Individuals. 

Why did we decide to engage in the difficult business of capital introduction (aka fundraising)? Two reasons: 

1. Because the vast majority of fund managers are good at investment analysis and portfolio management, but are terrible at  raising money, and;
2. AUM attracts more AUM

Tokenomics introduces a new tool that could be utilized to facilitate capital introduction. Besides having managers purchase tokens to pay fees and for arbitrage, as suggested, MLN tokens could be used to encourage new investors to try funds. I’m not an expert on tokenomics, but some ideas include: 

- what if Enzyme offered new investors MLN as an incentiveto their support its clients? 
- what if managers could allocate MLN tokens to investors as a performance enhancer to entice early adopters, perhaps the tokens would be vested over time. 
- what if investors who enjoy solid performance in their fund could refer other investors to that fund, and earn MLN in the process, thereby taking the manager completely out of the fundraising process in some cases. 

The possibilities seem endless.

In any case, once a fund achieves key AUM levels, new groups of investors gain authorization to participate. For example, many pensions have covenants that prevent their investing in a fund in which their investment will be greater than X% of the total AUM of such fund. For institutional investors that have minimum investments of $10m, it becomes nearly impossible for them to invest in a start-up fund manager. We need to scale that fund so that larger investors can participate.

This was an unfortunate obstacle for VanthedgePoint clients because our [research](https://emergingmanagermonthly.com/2007/02/07/vanthedge-point-releases-2006-performance-results/) showed that hedge funds tended to outperform their larger brethren in their first three years of operation. This meant that only high net worth individuals, who had greater flexibility, benefitted from their performance. But once our upstart clients gained momentum thanks to their superior performance, their AUM began to grow rapidly.

**Enzyme should seek to incentivize investors thereby helping clients achieve critical mass that increases the revenue generated by the protocol.** 

**2. Grow customer base** 

> I don’t believe that making fund initiation free will help Aum. It will only lead to more funds that, especially with a limited asset universe, are just more of the same. I’m Against making the fund free therefore, fund managers should have some skin in the game to prevent them from spamming the platform with useless funds. (@cojoq)

> Given that Melon’s core proposition has always been about empowering the long-tail, we feel that it would be wrong to stunt up-and-coming managers by providing such a barrier to entry. Our preference is to grow with our users. If they are successful, Melon will be successful.(@perhamgirl)

> Enzyme seem to and should skew towards attracting professional and aspiring professional managers. There are other platforms out there that target and would always be more appealing to the retail/experimental user. (@nottrunner)

EDIT: Nothing will attract more customers (funds) than AUM growth. 

Lots of small hedge funds with little AUM and/or weak performance will cause reputational damage to the Enzyme protocol. Don't make the investor experience cumbersome by forcing them to filter or sort through the database of available funds. Investors don't have that type of patience.

It seems that an appropriate solution would be to institute a minimum fund size, which would result in greater AUM-fee revenue for the protocol, and to eliminate the fee to create a new fund because of its de minimis impact. AUM growth is the key, not some tiny fee to create a fund. Enzyme must attract aspiring managers, particularly those who are willing to put their money where their mouth is by investing a significant amount of their own net worth in their fund.

In fact, this emerged as the most important due diligence item among investors we introduced to our clients. If a manager had invested only a little of his/her net worth in their own fund, then it would set off red flags to potential investors. Investors want to know that the person with whom they are about to entrust their money has the confidence to bet on themselves and their strategy.

**3. Grow protocol revenues**

> I don’t like the entry fees for investors (10bps). I’d rather see those free, yet the redemption fee higher, say 50bps. Incentive should be to get and keep as much Aum on the platform as possible. (@cojoq)

> Fees are paid for by the asset managers to help them run a business. (@dennygalindo)

> I think it would also be wise to consider tiered AuM-based fees so that they become very low as the fund becomes very big - that's typically what a traditional fund administrator would do. Something like the first $125M at 10bp, the next $125M at 7.5bp, the next $250M at 5bp, everything above that at 2.5bp?(@rusnewton)

> Perhaps you can add fees for different features & services. so if a fund manager want feature x s/he pays a monthly fee for that. I also think a fee to set up the fund is totally fine. I think the fund manager should be able to get a 0% loan which get automatically charged from the performance fee. So s/he has to pay the transaction cost but the extra melon fee s/he can pay from the future performance fee.(@MouyouMoos)

> Caveat: There has been some discussion over whether an AUM-based fee should be charged to the fund or to the manager. We believe this topic should be explored further, and it is not something we reached consensus on in our initial meetings on the topic.(@perhamgirl)

> It is worth exploring what other potential uses Melon Terminal might have beyond simply replacing the existing hedge fund administration layer in traditional finance.(@rusnewton )

A tiered AUM-based fee structure was and still is quite common. Serious managers accept that there are start-up costs, and that their investment performance will determine whether they could achieve critical mass or not. If they cannot do so within three years, then in the vast majority of cases the fund wound down regardless of its track record.

The idea to not charge an investor any fee to invest in a fund is a wise one, and to charge the investor a redemption fee is reasonable and could even be linked to the investor's duration in the fund. The redemption fee could be higher for a shorter duration and lower to no fee for a long duration in the fund. Perhaps it could be tied to some tokenomics incentive from when the investor joined the fund, which would lock up and vest MLN in some manner.

However, there are numerous other fees, and which entity — the fund or the manager — pays for the fees depends on the product or service. Trading, fund admin, legal and compliance, capital introduction, etc. fees are paid for by the fund, and operating expenses such as salaries, computers and rent, are paid for by the manager (management company). The manager typically has discretion over whether some fees, such as Bloomberg data, get charged to the fund or the management company.

As Enzyme expands its offering, I have no doubt that it could charge for numerous services just we did at VanthedgePoint. And when it does, it should provide managers with the ability to elect whether the fee for service XYZ should be charged to the fund or to the manager.

I hope you all find these suggestions useful. I plan to share more insights as I become more familiar with the protocol and the community including current and future users.

cheers,
geoff

**Comment by perhamgirl** [comment]
> **Enzyme should seek to incentivize investors thereby helping clients achieve critical mass that increases the revenue generated by the protocol.**

Hi Geoff, thanks so much for your feedback. It's nice to read from someone with your pedigree. My immediate takeaways from this are;

1. We should Explore AUM based incentives for depositors on the platform (possibly tied to length of time). Personally I’ve always been doubtful of this approach but you seem to have provided evidence-based research that we should probably revisit this more seriously. (Side note: I unfortunately was not able to access the research you supplied but have applied for a free trial period to Emerging Manager site and should have access to it soon pending their review process).  I should note though that we are very focused on growing AUM and I'm happy to talk to you about some of those initiatives offline.
2. We should find much better ways to aid decision making for investors between different Vaults (minimum AUM size, Manager investment into own Vault, etc). Assume zero patience.
3. In discussions with the Council, there was a general consensus that tier-based AUM fee is the way to go eventually. However, initially, the Council has decided to start with a flat 25bps fee for simplicity and until we get to critical mass. This is a good way to test the waters. We can and will revisit this parameter. Investment/Redemption fees have been put on hold for the moment. It was almost unanimously agreed that fee on investments cause unnecessary friction but we will certainly visit redemptions.
 
Regarding fees and the possibility of expensing them to a Fund or Vault - we’ll be introducing gas expensing in the next release where managers can claim back a large part of the gas incurred to the Vault for trading. We believe that costs like fund admin should become obsolete with Enzyme. Enzyme smart-contracts are meant to dis-intermediate the fund administrator role and largely compliance too. We still haven’t figured out a way to charge legal/cap intro feels to a Fund/Vault but maybe in the short term we can just accept that in the on-chain asset management world - costs are handled a little differently.
Very much enjoyed  reading your feedback and given your experience, would love to pick your brain further on some other ideas we've had to grow AUM - feel free to reach out to me [mona@avantgarde.finance](url) .
